backend/src/nodes/image_util_nodes.py

```diff
@@ -554,3 +554,32 @@ def run(
         ssim = round(calculate_ssim(comp_img, orig_img), 6)
 
         return (float(mse), float(psnr), ssim)
+
+@NodeFactory.register("chainner:image:canny_edge_detection")
+class CannyEdgeDetectionNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Detect the edges of the input image and output as black and white image."
+        self.inputs = [
+            ImageInput(),
+            NumberInput("Lower Threshold", minimum=0, default=100),
+            NumberInput("Higher Threshold", minimum=0, default=300),
```

Comment:
For grammatical context reasons, I would change 'Higher' to 'Upper', but other than that, the code itself looks good.